softwareComponents/kinematics/CMakeLists.txt

```diff
@@ -1,7 +1,13 @@
 cmake_minimum_required(VERSION 3.6)
 project(rofi)
 
-
 add_library(kinematics fReconfig.cpp)
 target_include_directories(kinematics INTERFACE .)
 target_link_libraries(kinematics PRIVATE legacy-configuration atoms)
+
+add_library(aabb include/aabb.hpp)
+target_link_libraries(aabb configuration)
```

Comment:
Přemýšlím, jestli by nestálo za to založit knihovnu `geometry` a postupně tam všechny věci stěhovat. A pak mít klidně knihovny jako `geometry::aabb` (v CMaku).  Důvod: začínáme mít spoustu věcí, kterou jsou spíše obecná geometrie a nesouvisí nutně s konfigurací nebo rekonfigurací.